Tests/Packet++Test/Tests/S7CommTests.cpp

```diff
@@ -0,0 +1,75 @@
+#include "../TestDefinition.h"
+#include "../Utils/TestUtils.h"
+#include "EndianPortable.h"
+#include "Packet.h"
+#include "S7CommLayer.h"
+#include "SystemUtils.h"
+
+PTF_TEST_CASE(S7CommLayerParsingTest)
+{
+	timeval time;
+	gettimeofday(&time, nullptr);
+	READ_FILE_AND_CREATE_PACKET(1, "PacketExamples/S7comm.dat");
+
+	pcpp::Packet S7CommLayerTest(&rawPacket1);
```

Comment:
nit: variable names should begin with lowecase and I'd rename it to: `s7CommLayer`